spec4b-optimizations/violation.tla

```diff
@@ -0,0 +1,362 @@
+---------------------------- MODULE counterexample ----------------------------
```

Comment:
How come we're including this one particular CE?

Reply:
removed. Good find!